about/package-scope.md

```diff
@@ -167,6 +159,34 @@ analyzing data.
  * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
 
 
+## Scope for packages that support analytics, statistics and modeling
+
+The mission of pyOpenSci's open peer review process is to:
+
+1. support improving the quality, usability and discoverability of maintained scientific Python software in support of open science.
+2. We also support maintainers in navigating the python packaging ecosystem.
+
+pyOpenSci is not a scientific journal. As such we are not able to review
+software packages that present novel or new models and statistical /
+analytics approaches without previous review of the analytical approach by a
+credible journal.
+
+We consider the following when determining whether an analytics-related package is within our review scope:
+
+1. If your package facilitates a scientist using a **known or vetted statistical, AI or  Analytical approach** we consider that in-scope. Before submitting to us, please ensure that your package's documentation directs users to existing paper(s) or pre-print(s) that document that approach's application. Further, be sure to link to these publications in your package review submission.
+
+The review for this package:
+
+* requires at least 1 domain specialist
+* will never vet the analytical method itself.
+
+2. If your package introduces a novel or newer analytic approach that is not yet known/ vetted/ accepted, we can not review it. We cannot review projects that exist as a proof-of-concept demonstration of a model or analytical approach that might accompany a paper. In this case, the approach should be sent to a scientific journal for vetting.
```

Comment:
```suggestion
2. If your package introduces a novel or newer analytic approach that is not yet vetted/ accepted by a scientific journal, we can not review it. We cannot review projects that exist as a proof-of-concept demonstration of a model or analytical approach that might accompany a paper. In this case, the approach should be sent to a scientific journal for vetting.
```
nit: "known" might be different for different people

Reply:
great catch!